src/core/address.test.js

```diff
@@ -152,4 +152,32 @@ describe("address", () => {
       expect(x).toEqual(input());
     });
   });
+
+  describe("toString and fromString", () => {
+    const examples = [mansion(), fakeMansion(), mattressStore()];
```

Comment:
Let's make this a function that generates the examples to avoid shared state across test cases:
```js
const examples = () => [mansion(), fakeMansion(), mattressStore()];
```